app/models/concerns/planning_application/validation_request.rb

```diff
@@ -31,9 +31,16 @@ module ValidationRequest
       end
     end
 
-    def validation_requests(post_validation: false)
-      (replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests)
-        .send(enumerable_method(post_validation), &:post_validation?).sort_by(&:created_at).reverse
+    def validation_requests(post_validation: false, include_description_change_validation_requests: false)
```

Comment:
This method's doing a lot now, and I've done my best but I'm sure it could be prettier! Suggestions very welcome 😅

Reply:
Not sure if it would be worth turning this stuff into a class perhaps. 

We can initialize it with what we need and then run methods in there? Perhaps if this needs any more logic down the line we do this. Or whatever the best OOP design is for this type of thing

Reply:
Yes, I like the sound of that! Let's say if it needs any further changes (and I feel like it may, given it's masquerading as an active record relation!) we will extract it out into its own thing